runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7712,8 +7712,9 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
       if ((_compInfo.getPersistentInfo()->getRemoteCompilationMode() == JITServer::CLIENT) &&
           TR::Options::canJITCompile())
          {
-         bool forcedLocal = false;
-         bool doLocalCompilation = entry->isAotLoad() || cannotDoRemoteCompilation || preferLocalComp(entry);
+	 TR_ResolvedMethod *resolvedMethod = fe->createResolvedMethod(&trMemory, (TR_OpaqueMethodBlock *)method);
+         bool forcedLocal = _compInfo.isJITServerRemoteExclude(resolvedMethod->signatureChars(),resolvedMethod->convertToMethod()->methodType());
```

Comment:
`resolvedMethod->signatureChars()` is the wrong API here. You need to call `resolvedMethod->signature(&trMemory)`. Look at other similar places.